project/Dependencies.scala

```diff
@@ -81,21 +81,22 @@ object Dependencies {
   )
 
   val akkaProjections = Seq(
-    "com.lightbend.akka" %% "akka-projection-core"         % AkkaProjections,
-    "com.lightbend.akka" %% "akka-projection-eventsourced" % AkkaProjections,
-    "com.lightbend.akka" %% "akka-projection-kafka"        % AkkaProjections,
-    "com.lightbend.akka" %% "akka-projection-cassandra"    % AkkaProjections,
-    "com.lightbend.akka" %% "akka-projection-jdbc"         % AkkaProjections,
-    "com.lightbend.akka" %% "akka-projection-testkit"      % AkkaProjections
+    "com.lightbend.akka" %% "akka-projection-core"          % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-eventsourced"  % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-durable-state" % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-kafka"         % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-cassandra"     % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-jdbc"          % AkkaProjections,
+    "com.lightbend.akka" %% "akka-projection-testkit"       % AkkaProjections
   )
 
   val akkaGrpc = Seq(
     // maven plugin brings in Scala dependencies that conflict with current crossVersion of BOM
     // `Modules were resolved with conflicting cross-version suffixes in ProjectRef(uri("file:/home/seglo/source/akka-platform-guide/akka-platform-dependencies/"), "akka-platform-dependencies") ...`
     // maven plugin versions can't be set with BOM/dependencyManagement
     //("com.lightbend.akka.grpc" % "akka-grpc-maven-plugin" % AkkaGrpc).withExclusions(Vector(ExclusionRule.everything)),
-    "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % AkkaGrpc,
-    "com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
+    "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % AkkaGrpc
+    //"com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
```

Comment:
```suggestion
    // TODO can akka-grpc-codegen be removed from bom?
    // not sure why, but akka-grpc-codegen is only released for Scala 2.12
    // (and I don't think it's supposed to be used via bom anyway?)
    //"com.lightbend.akka.grpc" %% "akka-grpc-codegen" % AkkaGrpc
```

Reply:
Publishing the codegen for 2.13 is definitely something we want, but broke when we switched to publishing to sonatype with sbt-ci-release (https://github.com/akka/akka-grpc/issues/1400).

I agree it doesn't make too much sense to have it in the bom, though.

Reply:
Thanks for confirming, I have removed it